tests/model/test_scene.py

```diff
@@ -151,5 +151,5 @@ def test_decode_scene_with_invalid_format():
   ]
 }"""
     comment_removed = "\n".join([re.sub("//.*", "", line) for line in json.split("\n")])
-    with pytest.raises(Exception):
+    with pytest.raises(RuntimeError):
```

Comment:
この変更では、[B017](https://docs.astral.sh/ruff/rules/assert-raises-exception/)の趣旨が満たされていません。
結局、なぞのRuntimeErrorをチェックしているだけだからです。

```suggestion
    with pytest.raises(Exception, match="kitti_velodyneのformatは.+である必要がありますが、hogehogeが指定されています"):
```